src/forcing/ForcingsEngineLumpedDataProvider.cpp

```diff
@@ -0,0 +1,134 @@
+#include "DataProvider.hpp"
+#include <chrono>
+#include <forcing/ForcingsEngineLumpedDataProvider.hpp>
+
+namespace data_access {
+
+using Provider     = ForcingsEngineLumpedDataProvider;
+using BaseProvider = Provider::base_type;
+
+std::size_t convert_divide_id_stoi(const std::string& divide_id)
```

Comment:
Is there an existing utility function elsewhere that already serves this purpose? I'm concerned about having a copy here that might need to change if the format of divide IDs changes.

Reply:
Ok, it's fine.